### ISSUE TITLE
[Lighthouse / Contrast Ratio / Helpful]

### ISSUE THREAD

**Issue by DJBunge** [google/WebFundamentals#4351]
I received the message, "Background and foreground colors have a sufficient contrast ratio" and followed the links. https://developers.google.com/web/tools/lighthouse/audits/contrast-ratio

The http://leaverou.github.io/contrast-ratio/#%23dcdcdc-on-%23212121 url is showing a value of 11.7.

I also installed and ran ChromeLens; it did not return the message.

**Comment by DJBunge** [comment]
This is for the site http://djbunge.com/ if you need to review the site itself.

**Comment by kaycebasques** [comment]
I haven't checked your site but I think that audit is buggy.

The footer elements on my blog fail the contrast test, yet it's literally white text on black background.

Repro: run LH on https://wordsmithweekly.com/ and look at the footer elements that fail the contrast ratio test

cc @paulirish @robdodson @brendankenny

**Comment by robdodson** [comment]
Yeah I think this is a bug in axe that we fixed but they haven't done a new release yet. https://github.com/GoogleChrome/lighthouse/issues/1615#issuecomment-285515599

I'll ping them directly today.

**Comment by kosamari** [comment]
closing this, referenced issue in lighthouse repo is fixed & closed :)
